### PR TITLE
Added TAC Meeting Notes For 2024-06-27

### PR DESCRIPTION
**Pull request by bensternthal**
@riaankleinhans

### CODE REVIEW

**Review by fritzalder — APPROVED** [review]

File: TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+In the last meeting the group discussed feedback on the CC summit.
+
+### PQC in CCC
+
+In the last meeting John Manferdelli discussed implementing PQC
+
+## Announcements
+
+Sal Kimmich (SK) stated that the mentorship program is now open and that mentorships can begin at anytime. SK asked interested folks to reach out to Sal or Yash.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md:99]:
```suggestion
Sal Kimmich (SK) stated that the mentorship program is now open and that mentorships can begin at anytime.
https://github.com/confidential-computing/governance/blob/main/mentoring/README.md
TAC members should contact their respective projects about this program.```

File: TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## TAC Objectives / TAC Goals: Ecosystem
+
+DM reviewed the TAC’s 2024 objectives and the three focus areas: Projects, Ecosystem, and Community. Alec Fernandez (AF) reviewed the 2024 objective for the GRC SIG. AF then discussed the three proposed Key Results. Discussion ensued.  Mark Novak (MN) suggested collaborating with other SIGs and groups, such as outreach, to help complete KRs. 
+
+MN shared the SIGs discussion of creating expectation patterns and shared a drive folder of [documents](https://drive.google.com/drive/u/0/folders/1n_wmIWPm3qwll78iRRIAzSGNXB_VdGwD). MN reviewed the first pattern they are developing: [Confidential Payload Governance](https://docs.google.com/document/d/1jzoEWmPCeAWV_g6kcJ3pzX9_UdljraM_2qVnwW8sT_s/edit). Discussion ensued. DM encouraged folks to join the GRC meeting and to leave feedback in the document. 
+
+## Announcements
+
+DM stated that the CCC is responding to three NIST documents. AF was unsure if these were the documents that were responded to.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md:111]: @riaankleinhans can you clarify about the NIST documents in the slides. Unclear if these are in progress and who has ownership or suggested them.

File: TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+   " + " indicates attending
+
+#### Other Attendees
+
+* Bokdeuk Jeong (Samsung)
+* Cosmin Aprodu
+* Dayeol Lee (TikTok)
+* Deli Gong
+* Henk Birkholz
+* Manu Foutaine (Hushmesh)
```
Comment (thomas-fossati) [inline TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md:57]:
```suggestion
* Manu Fontaine (Hushmesh)
```

File: TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md (hunk 2 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md:113]: It came in the TAC mailing list, sugesting the TAC could comment. link in the slides
https://lists.confidentialcomputing.io/g/tac/message/1327

**Pull request merged**